sirepo/job_supervisor.py

```diff
@@ -455,22 +455,22 @@ def _ops_to_cancel():
             jobCmd='compute',
             nextRequestSeconds=self.db.nextRequestSeconds,
         )
+        self.__db_init(req, prev_db=self.db)
+        self.db.pkupdate(
+            computeJobSerial=int(time.time()),
```

Comment:
@robnagler is it a problem that the computeJobSerial is now written before the job is going to be run (before all of the awaits)? I don't think so but I want to confirm.

Reply:
This is a bug. Definitely want computeJobSerial written right away, when the job is queued.

I thought we saved the time we queued. Technically that could be computeJobSerial, but maybe it should be a distinct value.